### DISCUSSION TITLE
host headers

### DISCUSSION THREAD

**Discussion by tbne0** [FlareSolverr/FlareSolverr#722] [Q&A]
Is it possible to run only in the same way as the example?

curl -L -X POST 'http://localhost:8191/v1' \
-H 'Content-Type: application/json' \
--data-raw '{
  "cmd": "request.get",
  "url":"http://www.site.com/",
  "maxTimeout": 60000
}'

Or can i also add the following options to execute it?

-k -H "Host: site.com" https://123.123.123.123

**Comment by ngosang** [comment]
No, it's not supported.
* If you are running from source code => You could edit your /etc/hosts
* If you are using Docker => https://stackoverflow.com/questions/29076194/using-add-host-or-extra-hosts-with-docker-compose
